control/getCategories.js

```diff
@@ -0,0 +1,12 @@
+// const model = require('seed/lib/seed/errors/model')
+const database = require('../models/getdb.js')
+
+games = function(req, res) {
```

Comment:
make sure to name functions descriptively, what is this function doing? Games is quite broad given that everything we are building is linked to the topic of board games, it will become confusing very quickly. Similarly, if a file is a controller or model, try to follow naming conventions such as categories.model.js and categories.controller.js so that it is apparent which is which